com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsControl.cs

```diff
@@ -544,14 +565,17 @@ private void OnDestroy()
 
         private void OnHostSelected(SelectEnterEventArgs args)
         {
+            isHostSelected = true;
```

Comment:
Does this need to be set to `false` in `OnHostDeselected`?

Reply:
> Does this need to be set to `false` in `OnHostDeselected`?

@keveleigh yes my silly mistake. should be fixed now